app/models/assignment.rb

```diff
@@ -957,12 +957,9 @@ def current_grader_data
       groups[[group_id, group_name, count]]
       groups[[group_id, group_name, count]] << ta unless ta.nil?
     end
-    # TODO: improve the group_sections calculation.
-    # In particular, this should be unified with Grouping#section.
     group_sections = {}
-    self.groupings.includes(:accepted_students).find_each do |g|
-      s = g.accepted_students.first
-      group_sections[g.id] = s&.section_id
+    self.groupings.include(:section).each do |g|
```

Comment:
@JunruL This should be `include`, not `includes`. This indicates that this method is not actually tested, since otherwise an error would have been raised.

So, in this PR please also add some tests for this model method. Note that this method returns a hash; you only need to write some tests for the `groups` key, and the `section` value inside each group. Writing new tests will take a bit of time, but is an excellent experience. My advice is to have some tests which create groupings with no members, groupings with inviters that don't belong to a section, and groupings that have inviters that do belong to sections.

Reply:
It seems that what I used was `include`. So do I need to use `includes` instead?

Reply:
Yes, that's correct. You can see the error when you write tests for this method, or you can find where this method is called from the UI.